### PR TITLE
[SEA]Jailer of Temperence, Faith, Fortitude / [SKY]God Add. Effects.

### PR DESCRIPTION
**Pull request by nesstea**

### CODE REVIEW

File: scripts/zones/Grand_Palace_of_HuXzoi/Zone.lua [hunk 1]
```diff
@@ -8,6 +8,7 @@ package.loaded["scripts/zones/Grand_Palace_of_HuXzoi/TextIDs"] = nil;
 
 require("scripts/globals/settings");
 require("scripts/zones/Grand_Palace_of_HuXzoi/TextIDs");
+require("/scripts/zones/Grand_Palace_of_HuXzoi/MobIDs");
```
Comment (TeoTwawki) [inline scripts/zones/Grand_Palace_of_HuXzoi/Zone.lua]: duplicate require

File: scripts/zones/Grand_Palace_of_HuXzoi/mobs/Eo_zdei.lua [hunk 2]
```diff
@@ -7,6 +7,7 @@
 -- Animation Sub 3 Ring Form
 -----------------------------------
 
+require("/scripts/zones/Grand_Palace_of_HuXzoi/MobIDs");
```
Comment (TeoTwawki) [inline scripts/zones/Grand_Palace_of_HuXzoi/mobs/Eo_zdei.lua]: leading slash not needed, just fyi. doesn't hurt it - still works.

Comment (nesstea) [inline scripts/zones/Grand_Palace_of_HuXzoi/mobs/Eo_zdei.lua]: Want me to remove it...? Another commit lol.

Comment (TeoTwawki) [inline scripts/zones/Grand_Palace_of_HuXzoi/mobs/Eo_zdei.lua]: I tend to just strip them as come across them figuring if you have to make another commit for something else anyway, may as well.

File: scripts/zones/Grand_Palace_of_HuXzoi/Zone.lua (hunk 1 above)
Comment (nesstea) [inline scripts/zones/Grand_Palace_of_HuXzoi/Zone.lua]: Look again

Comment (TeoTwawki) [inline scripts/zones/Grand_Palace_of_HuXzoi/Zone.lua]: yeah I see it. 1st and last letter brain trick.

File: scripts/zones/Grand_Palace_of_HuXzoi/mobs/Eo_zdei.lua (hunk 2 above)
Comment (nesstea) [inline scripts/zones/Grand_Palace_of_HuXzoi/mobs/Eo_zdei.lua]: It's done in a future commit.

File: scripts/zones/Grand_Palace_of_HuXzoi/Zone.lua [hunk 3]
```diff
@@ -105,4 +110,20 @@ function onEventFinish(player,csid,option)
...
         player:setVar("Hu-Xzoi-TP",0);
     end
+end;
+
+-----------------------------------
+-- onGameHour
+-----------------------------------
+
+function onGameHour(npc, mob, player)
+	local VanadielHour = VanadielHour();
```
Comment (santssoft) [inline scripts/zones/Grand_Palace_of_HuXzoi/Zone.lua]: onGameHour only called for Zones (zone.lua), not for NPCs or Mobs.

See luautils.cpp line 2778 for declaration of onGameHour

Comment (nesstea) [inline scripts/zones/Grand_Palace_of_HuXzoi/Zone.lua]: Ah yes. Changes will happen today.

Comment (nesstea) [inline scripts/zones/Grand_Palace_of_HuXzoi/Zone.lua]: Done https://github.com/nesstea/darkstar/commit/3c98539da2d743ce6512b6a6d055f8daa4680c47

**Comment by teschnei** [comment]
can you rebase out your merge commits and bug fix commits?

**Comment by nesstea** [comment]
Rebuilding from scratch.
